cdm_processing/cdm_processor.py

```diff
@@ -170,8 +170,12 @@ def _create_sequence_tables(self,
               "INNER JOIN person " \
               "  ON visits.person_id = person.person_id"
         con.execute(sql)
-        if self._settings.label_sub_folder is not None:
-            part1 = "  label, "
+        if self._settings.labels is not None:
+            label_columns = con.query("SELECT column_name FROM information_schema.columns WHERE table_name = 'labels'")
+            if "is_training" in label_columns.arrow()["column_name"].to_pylist():
+                part1 = "  is_training,  label, "
+            else:
+                part1 = "  label, "
```

Comment:
Added a new column to labels file, indicating whether the row is part of training or validation set. Necessary so  the PLP can control the split.